src/course-outline/page-alerts/messages.js

```diff
@@ -57,6 +57,34 @@ const messages = defineMessages({
     id: 'course-authoring.course-outline.page-alert.generic-error.description',
     defaultMessage: 'Unable to {actionName} {type}. Please try again.',
   },
+  newFileAlertTitle: {
+    id: 'course-authoring.course-outline.page-alert.paste-alert.new-files.title',
+    defaultMessage: 'New {newFilesLen, plural, one {file} other {files}} added to Files.',
```

Comment:
Pretty soon we'll start linting for messages that don't have descriptions.  It's an important part of i18n, as it makes the lives of translators much easier.  I'm not going to block PRs on this until https://github.com/openedx/frontend-build/pull/516 merges, but consider this friendly heads-up. :)